docs/serverless/README.md

```diff
@@ -0,0 +1,53 @@
+# Serverless Agent
+
+The Serverless Agent is a build of the Datadog Agent capable of running
+in serverless environments, to collect custom metrics, the logs generated by the
+function and the traces of the function execution. Today, its main usage is to
+build the Datadog Lambda Extension for AWS Lambda.
+
+## Concepts
+
+Today, the Serverless Agent is intended to run as a side-car of the serverless function.
+
+### Validate the presence of the Serverless Agent and order a flush
+
+The Serverless Agent runs an HTTP server on port 8124.
+
+With the Datadog Lambda Extension, this HTTP server listens for two
+different routes:
+    - /lambda/hello
+    - /lambda/flush
+
+These routes are not doing anything in specialized for AWS, however, they can be
```

Comment:
```suggestion
```

agree here, we can get rid of this whole sentence because the important info is in the next sentence.